samples/qa-dropdown/src/views/MainPanel.js

```diff
@@ -0,0 +1,26 @@
+import Button from '@enact/agate/Button';
+import Dropdown from '@enact/agate/Dropdown';
+import {Panel, Panels} from '@enact/agate/Panels';
+import {useCallback, useState} from 'react';
+
+const MainPanel = () => {
+	const [open, setOpen] = useState(false);
+
+	const handleOpen = useCallback(() => setOpen(true), []);
+	const handleClose = useCallback(() => setOpen(false), []);
+
+	return (
+		<Panels>
+			<Panel title="QA Sample - Dropdown">
+				<Dropdown onClose={handleClose} onOpen={handleOpen} open={open} size="large" title="language">
+					{['English', 'Korean', 'Spanish', 'Amharic', 'Thai', 'Arabic', 'Urdu', 'Simplified Chinese', 'Traditional Chinese', 'Vietnamese']}
+				</Dropdown>
+				<Button size="large">
+					Enter
+				</Button>
+			</Panel>
+		</Panels>
```

Comment:
```suggestion
		<Panels noCloseButton>
			<Panel title="QA Sample - Dropdown">
				<Header	hideLine title="QA Sample - Dropdown" />
				<Dropdown onClose={handleClose} onOpen={handleOpen} open={open} size="large" title="language">
					{['English', 'Korean', 'Spanish', 'Amharic', 'Thai', 'Arabic', 'Urdu', 'Simplified Chinese', 'Traditional Chinese', 'Vietnamese']}
				</Dropdown>
				<Button size="large">
					Enter
				</Button>
			</Panel>
		</Panels>
```
It's nicer to show a header for this sample. also to remove the closeButton of the panel:
![image](https://user-images.githubusercontent.com/63335068/191041365-9e1a1715-e904-4c30-be25-5b3b0e36525b.png)

Please try to also vertical align the dropdown and the button. thank you!